lib/chess/store/game.ex

```diff
@@ -22,20 +24,21 @@ defmodule Chess.Store.Game do
   """
   def create_changeset(struct, params \\ %{}) do
     struct
-    |> cast(params, [:board, :user_id])
+    |> cast(params, [:board, :user_id, :opponent_id])
     |> put_change(:board, Board.default)
-    |> validate_required([:board, :user_id])
+    |> validate_required([:board, :user_id, :opponent_id])
```

Comment:
I find it helpful to abstract this to `defp required_fields, do: ~w(board user_id opponent_id)a` and then use it in for casting and validating.